tests/unit/utils/test_log.py

```diff
@@ -255,6 +255,30 @@ def test_python_warnings_werror(self, args):
             warnings.warn("test warning", PendingDeprecationWarning)
 
 
+@pytest.mark.parametrize(
+    'console_cli,console_conf,console_expected,ram_conf,ram_expected',
+    [
+        (None, None, logging.INFO, None, logging.NOTSET),
+        (None, None, logging.INFO, 'CRITICAL', logging.CRITICAL),
+        (None, 'WARNING', logging.WARNING, 'INFO', logging.INFO),
+        ('INFO', 'WARNING', logging.INFO, 'VDEBUG', logging.VDEBUG),
+        ('WARNING', 'INFO', logging.WARNING, 'CRITICAL', logging.CRITICAL),
+    ])
+def test_init_from_config(mocker, console_cli, console_conf, console_expected,
+                          ram_conf, ram_expected):
+    args = argparse.Namespace(debug=False, loglevel=console_cli, color=True,
+                              loglines=10, logfilter="", force_color=False,
+                              json_logging=False, debug_flags=set())
+    log.init_log(args)
+
+    conf = mocker.Mock()
```

Comment:
Use the `config_stub` fixture instead (you can then do `config_stub.val.logging.level.ram = ram_conf` and most likely use `log.init_from_config(config_stub)`